test/serializers.test.js

```diff
@@ -70,7 +70,7 @@ test('req serializer', function (t) {
             log.info({req: theReq}, 'the request');
             var lastRecord = records[records.length-1];
             t.equal(lastRecord.req.method, 'GET');
-            t.equal(lastRecord.req.url, theReq.url);
+            t.equal(lastRecord.req.url, theReq.originalUrl);
```

Comment:
This won't pass since the tests are using a vanilla `http.createServer`. If you want to test that `req.originalUrl` is being used then you might want to add another test and extend the request object with a `originalUrl` property, like express does.